client/payment-details/dispute-details/dispute-notice.tsx

```diff
@@ -35,34 +36,81 @@ const DisputeNotice: React.FC< DisputeNoticeProps > = ( {
 			'woocommerce-payments'
 		);
 
-	/* translators: <a> link to dispute documentation. %s is the clients claim for the dispute, eg "The cardholder claims this is an unrecognized charge." */
-	let noticeText = __(
-		'<strong>%s</strong> Challenge the dispute if you believe the claim is invalid, ' +
-			'or accept to forfeit the funds and pay the dispute fee. ' +
-			'Non-response will result in an automatic loss. <a>Learn more about responding to disputes</a>',
-		'woocommerce-payments'
-	);
-	let learnMoreDocsUrl =
-		'https://woocommerce.com/document/woopayments/fraud-and-disputes/managing-disputes/#responding';
+	// Format the deadline date
+	const dueByDate = moment
```

Comment:
I'd suggest to use this utility function [formatDateTimeFromTimestamp](https://github.com/Automattic/woocommerce-payments/blob/18aacf2781a84594bece97367c9661d0c810f20f/client/utils/date-time.ts#L59). Using it, the date and time will be rendered using the `Date Format` and `Time Format` settings the user saves in `Settings > General`. Something like this:

```suggestion
	const dueByDate = formatDateTimeFromTimestamp( dispute.evidence_details?.due_by ?? 0 )
```

Reply:
applied here 3b8a463383c9c45672001a0d5a9d4f4348842fc3